---a/net/minecraft/inventory/IInvBasic.java
+++b/net/minecraft/inventory/IInvBasic.java
@@ -1,7 +1,7 @@
 package net.minecraft.inventory;
 
-public interface IInvBasic
-{
+public interface IInvBasic {
+
     void onInventoryChanged(InventoryBasic p_76316_1_);
-}
+}
